docs/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Purpose
+_Briefly describe the purpose of the change, and/or link to the JIRA ticket for context_
+
+Fixes LPA-####
+
+## Approach
+
+_Explain how your code addresses the purpose of the change_
+
+## Learning
+
+_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Refunds service_
+
+## Checklist
+
+* [ ] I have performed a self-review of my own code
+* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
+* [ ] I have added tests to prove my work
+* [ ] The product team have tested these changes
```

Comment:
Phrasing here might need a tweak? It has been demoed to product manager?

Reply:
We're yet to agree as a team what this template should include, I'm getting a headstart. There's things we can agree to and reference here about our agreed best practice for testing, where docs should go, and how we get acceptance.